unit/ctest_position_map.c

```diff
@@ -441,7 +441,7 @@ test_position_polynomial_map_optimize_1x()
   pos_map->bmag_ctx->crange_global = &localRange;
   pos_map->bmag_ctx->cbasis = &basis;
   pos_map->bmag_ctx->cgrid = &grid;
-  gkyl_array_copy(pos_map->bmag_ctx->bmag, bmag_global);
+  gkyl_array_set(pos_map->bmag_ctx->bmag, 1.0, bmag_global);
```

Comment:
i think this is tied up in another PR, so is this what you still want to do here or do you want to correct it here too?